### PR TITLE
Moves auto unlocked exofab nodes to techweb nodes

### PR DESCRIPTION
**Pull request by 81Denton**
:cl: Denton
tweak: Techweb nodes that are available to exofabs by roundstart have been moved to basic research technology.
balance: Ripley APLU circuit boards are now printable by roundstart.
balance: Odysseus, Gygax, Durand, H.O.N.K. and Phazon mech parts have to be researched before becoming printable (same as their circuit boards).
/:cl:

Closes: #39293

Exofabs start with nodes unlocked by roundstart, which leads to the awkward situation where they can print cyborg parts, but the useless "cyborg parts" node still exists and has to be researched as a prerequisite for other nodes.

Same goes for mech nodes - you can print all mech parts, but not finish one of them since the circuit printer doesn't have mech nodes unlocked.

I removed `node_autounlock_ids` from the exofab and instead set `starting_node = TRUE` for MMIs, cyborg parts, basic mech technology and basic mech equipment.
I did not include Odysseus, Gygax, Durand, H.O.N.K. and Phazon mech parts, since that would allow printing the complete mech by roundstart.

**tl;dr** Roboticists gain Ripley boards, but lose Odysseus/Gygax/Durand/H.O.N.K./Phazon mech parts roundstart. 
As a consequence, this slightly lowers the total cost for researching nodes that had the above mentioned ones as prereqs.

### CODE REVIEW

**Comment by zxaber** [comment]
Yes please

**Comment by kevinz000** [comment]
hmm might have side effects but whatever

**Comment by ExcessiveUseOfCobblestone** [comment]
Thanks, I'm totally against having arbitrary nodes that do nothing.
